superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -365,12 +375,19 @@ class Header extends React.PureComponent {
     this.setState({ showingReportModal: false });
   }
 
+  handleReportModalclick() {
+    const attachedReportExists = this.props.report.count > 0;
```

Comment:
could report be null or undefined?

Reply:
it can be undefined, in the toggle dropdown ticket I actually added in logic for that

Reply:
Ok, then we should use optional chaining here in case

Reply:
added